app/assets/javascripts/discourse/app/templates/components/details-under-link.hbs

```diff
@@ -0,0 +1,9 @@
+{{#unless showDetails}}
+  {{d-button action=(action "showDetails")
```

Comment:
nitpicking here but as you are new to the code base, the correct indentation here is:

```
{{d-button
  action=(action "showDetails")
  class="btn-link"
  label=label
  aria-label=label
}}
```